cmd/registry/cmd/util/util.go

```diff
@@ -30,3 +33,17 @@ func TargetConfiguration() (name string, c config.Configuration, err error) {
 
 	return
 }
+
+// FQName ensures the project and location, if available,
+// are properly included in the resource name.
+func FQName(c connection.Config, name string) string {
```

Comment:
Could we make this a member function of connection.Config? I think the `util` package might attract some [negative attention](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_avoid_package_names_like_base_common_or_util) and there's not very much in it -- I think `NoActiveConfigurationError` also makes sense as `connection.NoActiveConfigurationError`

Reply:
What's funny is that I did have this as a member of connnection.Config in an earlier iteration and moved it out. I don't actually remember why exactly, other than it seems like maybe I thought it was more closely tied to cmd stuff. Whatever. I'll move it back. :)

Reply:
Oh, as for moving NoActiveConfigurationError error (and, I assume, the TargetConfiguration function?), that seems out of scope. I'll add an issue.